torch/_inductor/codecache.py

```diff
@@ -1622,38 +1666,61 @@ def _pad_to_alignment(raw_bytes: bytes) -> bytes:
             if config.aot_inductor.force_mmap_weights:
                 use_mmap_weights = True
 
-            object_build_options = CppTorchDeviceOptions(
+            compile_command: dict[str, Any] = {
+                "aot_mode": graph.aot_mode,
+                "device_type": device_type,
+                "use_mmap_weights": use_mmap_weights,
+                "use_relative_path": config.is_fbcode(),
+                "vec_isa": picked_vec_isa,
+            }
+            # If we're packaging via CMake, we build the whole code at max optimization.
+            wrapper_build_options = CppTorchDeviceOptions(
+                **compile_command,
+                min_optimize=not config.aot_inductor.package_cpp_only,
+                compile_only=True,
+            )
+            kernel_build_options = CppTorchDeviceOptions(
                 vec_isa=picked_vec_isa,
                 device_type=device_type,
                 aot_mode=graph.aot_mode,
                 compile_only=True,
```

Comment:
We actually need to keep `use_relative_path=config.is_fbcode()`.

Reply:
@desertfire That's entirely on me, this was intended to share usage of the `compile_command` dict defined above. I'll push the fix momentarily.